cmd/credentials.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/asaskevich/govalidator"
+	"github.com/go-openapi/strfmt"
+	"github.com/manifoldco/go-manifold"
+	"github.com/manifoldco/promptui"
+	"github.com/urfave/cli"
+
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/manifoldco/grafton/generated/connector/client"
+	"github.com/manifoldco/grafton/generated/connector/client/o_auth"
+	"github.com/manifoldco/grafton/generated/connector/models"
+)
+
+const passwordMask = '●'
+const apiURL = "https://api.%s.manifold.co/v1"
+
+func init() {
+	cmd := cli.Command{
+		Name:  "credentials",
+		Usage: "Rotates OAuth credentials for Manifold.co",
+		Subcommands: []cli.Command{
+			{
+				Name:   "rotate",
+				Usage:  "Creates a new credential and set the old one to expire in 24h",
+				Action: createCredentialsCmd,
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "provider",
+						Usage: "The label of the provider",
+					},
+					cli.StringFlag{
+						Name:  "product",
+						Usage: "The label of the product",
+					},
+				},
+			},
+		},
+	}
+
+	cmds = append(cmds, cmd)
+}
+
+func createCredentialsCmd(cliCtx *cli.Context) error {
+	ctx := context.Background()
+
+	providerLabel := cliCtx.String("provider")
+	if providerLabel == "" {
+		return cli.NewExitError("--provider flag missing", -1)
+	}
+
+	productLabel := cliCtx.String("product")
+	if productLabel == "" {
+		return cli.NewExitError("--product flag missing", -1)
+	}
+
+	p := promptui.Prompt{
```

Comment:
I think you should include a message that says the user is logging in to their manifold account